src/unix/udp.c

```diff
@@ -860,15 +977,7 @@ static int uv__setsockopt_maybe_char(uv_udp_t* handle,
 
 
 int uv_udp_set_broadcast(uv_udp_t* handle, int on) {
-  if (setsockopt(handle->io_watcher.fd,
-                 SOL_SOCKET,
-                 SO_BROADCAST,
-                 &on,
-                 sizeof(on))) {
-    return UV__ERR(errno);
-  }
-
-  return 0;
+  return uv__setsockopt(handle, SOL_SOCKET, SO_BROADCAST, &on, sizeof(on));
```

Comment:
This... is not the same... is it? You'll end up with `setsockopt(fd, IPPROTO_IP, SOL_SOCKET, &on, sizeof(on)` or `setsockopt(fd, IPPROTO_IPV6, SO_BROADCAST, &on, sizeof(on))`

Reply:
wow, derp